### DISCUSSION TITLE
我被诈骗了，这该判什么罪？！

### DISCUSSION THREAD

**Discussion by jiangshixie** [share121/inter-knot#3398] [其他]
<img src="https://img-static.mihoyo.com/communityweb/upload/8b198c0a467614f263e08593bd4b000d.png" alt="安比_无语" width="75" >今天我像日常研究邦布的构造，旁边突然弹出一个链接“点我看可爱邦布录像”，我顺手点了进去，没想到服务器不稳定让我加载了整整5分钟！我点开了视频，结果播放的是治安局最新的发诈骗宣传片！根本不是什么邦布啊喂！接着我又看到下面出现了另一个链接，上面写的是“点我看治安局防诈宣传片”，于是我又点了一下，发现……一个年轻男子在麦克风前跳舞，而且播放着轻快的音乐！这是什么诈骗手法啊！？

**Comment by jiangshixie** [comment]
@ZhuYuan-148

**Comment by Wax1234567** [comment]
不知道，好像是最近突然出现的那群自称“假面愚者”的人干的

**Comment by ZhuYuan-148** [comment]
前几天治安局的同事也有在网上冲浪的时候被相同手法骗了，点进去都是一样的视频，目前无法得知传播意义，但希望大家不要轻易点进未知连接！
